src/assisted_test_infra/download_logs/resources/man_sosreport.sh

```diff
@@ -1,135 +1,15 @@
 #!/bin/bash
-export LANG=C
 
-# If this script hangs, un-comment the below two entries and note the command that the script hangs on.  Then comment out that command and re-run the script.
-# set -x
-# set -o verbose
+SOS_TMPDIR="/var/tmp"
 
-[[ -d /tmp/sosreport ]] && rm -rf /tmp/sosreport
-mkdir /tmp/sosreport && cd /tmp/sosreport && mkdir -p  var/log etc/lvm etc/sysconfig network storage sos_commands/networking
+# cleanup any previous sos report
+sudo find "${SOS_TMPDIR}" -maxdepth 1 -name "sosreport*" -type f -delete
 
-echo -e "Gathering system information..."
-hostname &> hostname
-cp -a /etc/redhat-release  ./etc/ 2>> error_log
-uptime &> uptime
+sudo toolbox sos report --batch --tmp-dir "${SOS_TMPDIR}" --compression-type xz --all-logs \
+                   -o processor,memory,container_log,filesys,logs,crio,podman,openshift,openshift_ovn,networking,networkmanager,rhcos \
+                   -k crio.all=on -k crio.logs=on \
+                   -k podman.all=on -k podman.logs=on \
+                   -k openshift.with-api=on
```

Comment:
I saw there's an option to set a global timeout for all plugins
Maybe we should do so to make sure we always have as much data as we can?